openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -517,3 +520,17 @@ def _validate_email(email):
         raise AccountEmailInvalid(
             u"Email '{email}' format is not valid".format(email=email)
         )
+
+
+def set_has_profile_image(username, has_profile_image=True):
```

Comment:
My discomfort with this method is that it allows anyone to set the "has_profile_image" bit, which goes against the model of what we have tried to do with the other API methods. It's only here for the image uploading code, so I'd like to hide it. Honestly, I would prefer it lived in the image uploading package if that is feasible (using your trick to avoid the imports?).